### PR TITLE
added download counter, anime, and xiaro command

### PR DESCRIPTION
**Pull request by sourTaste000**
added download counter, anime, and xiaro command

### CODE REVIEW

**Comment by sourTaste000** [comment]
someone review my code lul

**Comment by sourTaste000** [comment]
also i need to change the channel id

File: main.js [hunk 1]
```diff
@@ -91,7 +93,29 @@ client.on("ready", () => {
...
+    setInterval(() => {
+        fetch("https://api.github.com/repos/kami-blue/nightly-releases/releases", {headers: {Authorization: `token ${auth.githubtoken}`}})
+            .then(response => response.json())
+            .then(data => {
+                const final = JSON.parse(JSON.stringify(data));
+                let j = 0;
+                for(let i = 0; i <= 29; i++){
+                    j += final[i].assets[0].download_count;
+                    console.log(j);
+                }
```
Comment (dominikaaaa) [inline main.js:110]: `downloadCount * (total / 30)` remember we should be doing some sort of math like so, with total being the total amount of releases in the repo. I'm not sure how to get the total as the release API returns the first 30.

I could have an API endpoint on kamiblue.org that automatically updates with the total?

Comment (sourTaste000) [reply main.js:110]: yes that would be great lol

Comment (sourTaste000) [reply main.js:110]: actually this line
```js
for(let i = 0; i <= 29; i++){
                    j += final[i].assets[0].download_count;
                    //console.log(j);
                }
```
calculates the total

Comment (dominikaaaa) [reply main.js:110]: this doesn't calculate the real total, because it only works for the last 30 releases.

Comment (sourTaste000) [reply main.js:110]: bruh ok

Comment (dominikaaaa) [reply main.js:110]: I'll make an API endpoint that gives you the total to do the math with

File: main.js [hunk 2]
```diff
@@ -85,13 +87,44 @@ client.on("ready", () => {
...
+     * @since  8/12/2020
+     */
+    setInterval(() => {
+        fetch("https://api.github.com/repos/kami-blue/nightly-releases/releases", {headers: {Authorization: `token ${auth.githubtoken}`}})
+            .then(response => response.json())
+            .then(data => {
+                const nightly = JSON.parse(JSON.stringify(data));
+                let nightlyCount = 0;
+                for(let i = 0; i <= 29; i++){ nightlyCount += nightly[i].assets[0].download_count }
+                fetch("https://api.github.com/repos/kami-blue/nightly-releases/releases", {headers: {Authorization: `token ${auth.githubtoken}`}})
```
Comment (dominikaaaa) [inline main.js:108]: this is supposed to be client

Comment (sourTaste000) [reply main.js]: 1eecc88

**Pull request merged**